docs/doc_needed/doc_control.lua

```diff
@@ -0,0 +1,103 @@
+-- use case scenario:
+-- intrspection.json --> contains all the statements present in src/lua/zencode_*.lua
+-- documented.txt    --> contains all the statements present in docs/_media/examples/zencode_cookbook/**/*.zen and in docs/_media/examples/zencode_cookbook/*.zen
+-- lua doc_control.lua introspection.json documented.txt
+
+-- return a table containing all the scenarios in alphabetic order
+-- and a table containg all the zencode statement divided by scenario
+local function all_statements(file_name)
+   io.input(file_name)
+   local d = {}
+   local scenarios = {}
+   local scenario, line
+   
+   while true do
+      line = io.read()
+      if not line then break end
+      
+      line = string.gsub(line, "^%s*(.-)%s*$", "%1")
+      if line == "{" then
```

Comment:
There is no certainty that the file json will be generated with a '{' on a single line... Is it sufficient to use a custom format like
```
scenario
statement
statement
statement

scenario
statement
....
```